docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,486 @@
+   | `uid_operator_key` | `string` | n/a | yes | The UID2 operator key, which you received as part of [UID2 Operator Account Setup](#uid2-operator-account-setup). |
+   | `uid_deployment_env` | `string` | `integ` | yes | Valid values: `integ` for integration environment, `prod` for production environment.<br/>Machine type is determined by the deployment environment: `integ` uses `n2d-standard-2` and `prod` uses `n2d-standard-16`. |
+
+1. (Optional, strongly recommended) Set the load balancer to HTTPS. Provide values for the parameters shown in the following table:
+
+   | Name | Type | Default | Required | Description |
+   | :--- | :--- | :--- | :--- | :--- |
+   | `ssl` | `bool`  | `false`| no | To set the load balancer to use HTTPS, which is recommended, set this flag to `true`.<br/>If you're using HTTPS you must also specify values for the `certificate` and `private_key` parameters. |
+   | `certificate` | `string`  | n/a | no | The contents of the HTTPS certificate. The certificate should be in PEM format.<br/>For example: `file('path/to/certificate.pem')`.<br/>Required if `ssl` is set to `true`.<br/>For details, see [google_compute_ssl_certificate](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_certificate#certificate) in the Terraform documentation. |
+   | `private_key` | `string`  | n/a | no | The contents of the private key for the HTTPS certificate. The private key should be in PEM format.<br/>For example: `file('path/to/private_key.pem')`. <br/>Required if `ssl` is set to `true`.<br/>For details, see [google_compute_ssl_certificate](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_certificate#private_key) in the Terraform documentation. |
+   
+1. (Optional) Provide parameter names and values for the additional input parameters shown in the following table. These parameters are always optional, but you might want to modify from the default values to better suit your requirements.
+
+   | Name | Type | Default | Required | Description |
+   | :--- | :--- | :--- | :--- | :--- |
+   | `region` | `string` | `us-east1` | no | The region that you want to deploy to. For a list of valid regions, see [Available regions and zones](https://cloud.google.com/compute/docs/regions-zones#available) in the Google Cloud documentation.<br/>NOTE: The UID2 Private Operator implementation for GCP Confidential Space is not supported in these areas: Europe, China. |
+   | `network_name` | `string` | `uid-operator` | no | The VPC resource name (also used for rules/ instance tags). |
+   | `min_replicas` | `number` | `1` | no | Indicates the minimum number of replicas you want to deploy. Default value: `1`. |
```

Comment:
Do we need to repeat the default value in the description given there is a already a column for it?